lightkurve/interact.py

```diff
@@ -299,14 +312,13 @@ def show_interact_widget(tpf, lc=None, notebook_url='localhost:8888', max_cadenc
                   "you can install bokeh using e.g. `conda install bokeh`.")
         return None
 
-    if lc is None:
-        lc = tpf.to_lightcurve(aperture_mask=tpf.pipeline_mask)
-    else:
-        if len(lc.time) != len(tpf.time):
-            log.error("The custom lightcurve provided to interact() does not contain "
-                      "the same number of cadences ({}) as the target pixel file ({})."
-                      "".format(len(lc.time), len(tpf.time)))
-            return None
+    if aperture_mask is None:
+        aperture_mask = tpf.pipeline_mask
```

Comment:
I recommend removing the above two lines in favor of:
```aperture_mask = tpf._parse_aperture_mask(aperture_mask)```

This will enable all the traditional masks to be supported.  To ensure that the pipeline mask is the default, change `aperture_mask=None` into `aperture_mask='pipeline'` in the function signature.